src/extension/byok/node/openAIEndpoint.ts

```diff
@@ -82,18 +80,27 @@ export class OpenAIEndpoint extends ChatEndpoint {
 	}
 
 	override createRequestBody(options: ICreateEndpointBodyOptions): IEndpointBody {
-		options.ignoreStatefulMarker = false;
```

Comment:
Moving this to `makeChatRequest2` doesn't work because then it gets overridden by the `super` call, I can fix it